common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -316,7 +316,8 @@ def export(self):
 
                     for s in seqs:
                         i += 1
-                        display_name = s.display_name.replace('&', '&amp;')
+                        name = s.display_name or s.display_name_with_default_escaped
+                        display_name = name.replace('&', '&amp;')
```

Comment:
Do we need a more comprehensive HTML encoding? (For things like `>`, etc?)

Reply:
I think, yes. I'll investigate this and test. Thanks for your point.

Reply:
I think, yes. I'll investigate this and test. Thanks for your point.